csharp/src/Google.Protobuf.Conformance/Program.cs

```diff
@@ -72,24 +72,21 @@ private static ConformanceResponse PerformRequest(ConformanceRequest request, Ty
                 switch (request.PayloadCase)
                 {
                     case ConformanceRequest.PayloadOneofCase.JsonPayload:
+                        JsonParser parser;
```

Comment:
This can be unconditional:

```csharp
bool ignoreUnknownFields = request.TestCategory == global::Conformance.TestCategory.JsonIgnoreUnknownParsingTest;
JsonParser parser = new JsonParser(new JsonParser.Settings(20, typeRegistry).WithIgnoreUnknownFields(ignoreUnknownFields));
```

Reply:
Applied -- thank you!